NEWS.md

```
@@ -1,10 +1,11 @@
+To read all history of releases go to the latest [release notes](https://docs.pgrouting.org/latest/en/release_notes.html)
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)